src/mlpack/tests/matrix_completion_test.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file matrix_completion_test.cpp
+ * @author Stephen Tu
+ *
+ * Tests for matrix completion
+ */
+#include <mlpack/core.hpp>
+#include <mlpack/methods/matrix_completion/matrix_completion.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "old_boost_test_definitions.hpp"
+
+using namespace mlpack;
+using namespace mlpack::matrix_completion;
+
+BOOST_AUTO_TEST_SUITE(MatrixCompletionTest);
+
+BOOST_AUTO_TEST_CASE(GaussianMatrixCompletionSDP)
```

Comment:
Yeah, I guess I am hoping for a comment on where the data comes from (randomly generated is fine, but presumably you did it according to some distribution), which might help someone later on when they are debugging the test.  What the test is actually doing is pretty obvious in my opinion, so I don't think there's a need to write a big long essay on what the purpose of the test is or anything.